WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -151,6 +151,13 @@ class AccountSettingsService {
         return notificationObserver.map(getSettings).startWith(getSettings())
     }
 
+    func primarySiteNameForSettings(settings: AccountSettings?) -> String {
```

Comment:
@koke the reason why that method has an optional is that `mapViewModel`'s [signature uses an optional as well, and this is consumed by that helper](https://github.com/wordpress-mobile/WordPress-iOS/blob/issue/4822-primary-blog-selector/WordPress/Classes/ViewRelated/Me/AccountSettingsViewController.swift#L39).

I'd be happy to kill the optional, but this leads me to a cross question for you: why is `mapViewModel` relying on an optional model?